sqlglot/parser.py

```diff
@@ -1852,11 +1856,13 @@ def _parse_column(self):
                 this = self.expression(exp.Dot, this=this, expression=field)
             this = self._parse_bracket(this)
 
-        return this
+        return self._attach_comment(this, self._prev.comment)
 
     def _parse_primary(self):
         if self._match_set(self.PRIMARY_PARSERS):
-            return self.PRIMARY_PARSERS[self._prev.token_type](self, self._prev)
+            return self._attach_comment(
```

Comment:
i'm not sure if this is the right place to put it, you can just put it in each primary parser?

Reply:
What were your concerns? I think it's less code-duplication if we have it here instead of each primary parser separately.